templates/http/docs/service.twig

```diff
@@ -0,0 +1,41 @@
+{{ method.method | caseUpper }} {{ method.path }} HTTP/1.1
+Host: {{spec.endpoint}}
+{% for key, header in method.headers %}
+{{ key }}: {{ header }}
+{% endfor %}
+{% for key,header in spec.global.defaultHeaders %}
+{{key}}: {{header}}
+{% endfor %}
+{% for header in spec.global.headers %}
+{% if header.key != 'Mode' and header.key != 'Locale' %}
+{{header.name}}: {% for node in method.security %}{% for key,x in node|keys %}{% if x == header.key %}{{node[x]['x-appwrite']['demo']}}{% endif %}{% endfor %}{% endfor %}
+
+{% endif %}
+{% endfor %}
+
+{% for key, header in method.headers %}
+{% if header == 'application/json' %}
+{% if method.parameters.body %}
+json: {
+{% for parameter in method.parameters.body %}
+  "{{ parameter.name }}": {{ parameter | paramExample }}{% if not loop.last %},{% endif %}
+
```

Comment:
Is this extra spacing required here?

Reply:
Without it the JSON gets formatted wierdly like so:
```json
json: {
"email": email@example.com,"password": password}
```
with it, it'll get formatted correctly like so: 
```json
json: {
  "email": email@example.com,
  "password": password
}
```
Correctly adding indentation and new lines, if I'm missing a better way of doing it please let me know 👍